pages/proposal.tsx

```diff
@@ -24,6 +24,37 @@ const Proposal = () => {
 				},
 			})
 
+			for (const [i, proposal] of proposalDetail.entries()) {
+				const proposalVotes = await near.nearViewFunction({
+					contractName: CONTRACT.DAO,
+					methodName: 'get_proposal_votes',
+					args: {
+						id: proposal.id,
+						from_index: 0,
+						limit: 10,
+					}
+				})
```

Comment:
kalo di page proposal keknya ga perlu ini sih, gaada nampilin list of votesnya soalnyaa